src/sync/incomingSyncUtils.js

```diff
@@ -632,6 +640,27 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
       numberSequence.addNumberToReuse(numberToReuse);
       break;
     }
+    case 'Pref': {
+      const { item } = record;
+      if (item === 'store_preferences') {
+        try {
+          const recordData = JSON.parse(record?.data);
+          const recordKeys = Object.keys(recordData);
+          recordKeys.forEach(key => {
+            const prefData = JSON.stringify(recordData[key] ?? {});
+            internalRecord = { id: key, data: prefData };
+            database.update(recordType, internalRecord);
+          });
+        } catch (error) {
+          // Silently ignore malformed prefs.
+        }
+      }
+      break;
+    }
+    case 'PrefBlob': {
+      // Ignore pref_blob records.
+      break;
+    }
```

Comment:
Is this for being explicit that we ignore pref blobs? I'm thinking it might be a little nicer to just leave it out

Reply:
Yeah explicit for now but also just a placeholder for later. Did the work and didn't think it was worth removing. Currently waiting on Adrian for info on any pref blobs we might need to sync. Seems like you and @andreievg on the same page on this one so will go ahead and remove!

Reply:
615d1b2a42f7545707699d21d693733763b1c32e.